### PR TITLE
Add enhanced formatting for Slack messages from Chattermill agent

### PR DESCRIPTION
**Pull request by duboff**

### CODE REVIEW

**Comment by duboff** [comment]
@rrodrigu3z can you pls review

File: app/models/agents/chattermill_response_agent.rb [hunk 1]
```diff
@@ -199,10 +199,20 @@ def handle(data, event = Event.new, headers)
     def send_slack_notification(response, event)
       link = "<https://huginn.chattermill.xyz/agents/#{event.agent_id}/events|Details>"
       parsed_body = JSON.parse(response.body) rescue ""
```
Comment (rrodrigu3z) [inline app/models/agents/chattermill_response_agent.rb:201]: Are we going to use the response body in the bot message?

**Comment by rrodrigu3z** [comment]
Hi @duboff, I pushed a little fix and updated the spec. Looks good, but I don't see the response body in the attachment, but I don't know if that is the desired behavior.

File: app/models/agents/chattermill_response_agent.rb (hunk 1 above)
Comment (duboff) [reply app/models/agents/chattermill_response_agent.rb:201]: You are right, will amend

**Comment by duboff** [comment]
@rrodrigu3z should be good to go, can you pls 🚢 ?

**Pull request merged**